services/scanners/task_parser.go

```diff
@@ -213,16 +218,34 @@ func (p *ParserTask) parseBlockTransactions(block oasis.Block) (err error) {
 			receiver = raw.Body.Account.String()
 		}
 
+		reclaimAmount := quantity.NewQuantity()
+		//Get reclaim amount
+		if txType.Type() == dmodels.TransactionTypeReclaimEscrow {
+			stakeAccount, err := p.stakingAPI.Account(p.ctx, &stakingAPI.OwnerQuery{
+				Height: block.Header.Height,
+				Owner:  raw.Body.Account,
+			})
+			if err != nil {
+				return err
+			}
+
+			reclaimAmount, err = stakeAccount.Escrow.Debonding.StakeForShares(&raw.Body.Shares)
+			if err != nil {
+				return err
+			}
+		}
+
 		dTxs[i] = dmodels.Transaction{
 			BlockLevel: uint64(block.Header.Height),
 			BlockHash:  hex.EncodeToString(block.Hash),
 			Hash:       tx.Hash().String(),
 			Time:       block.Header.Time,
 			//Same field
 			//Todo probably merge to single field
```

Comment:
Yeah, it would make sense to merge the 2 fields together.

`EscrowReclaimAmount` could also be merged with these 2.